multi-educator/src/Dscp/MultiEducator/Web/Educator/Types.hs

```diff
@@ -0,0 +1,31 @@
+-- | API handlers specific for multi educator.
+
+module Dscp.MultiEducator.Web.Educator.Types
+       ( CertificateName (..)
+       ) where
+
+import Data.List (span)
+import Servant (FromHttpApiData (..), ToHttpApiData (..))
+import System.FilePath (splitExtension)
+
+import Dscp.Core.Foundation
+import Dscp.Util
+
+data CertificateName = CertificateName
+    { cnEducatorId    :: Text
+    , cnCertificateId :: Id CertificateMeta
+    } deriving (Show, Eq, Generic)
+
+instance ToHttpApiData CertificateName where
+    toUrlPiece (CertificateName eId cId) =
+        toBase64Url @ByteString (encodeUtf8 $ eId <> ":" <> toHex cId) <> ".pdf"
```

Comment:
It looks suspicious that `toBase64Url` is used while `fromBase64Url` is not.

Reply:
:thinking: :thinking: :thinking:

Reply:
Fixed